src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,132 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_userAgeLess18YearsOld_notOk() {
```

Comment:
```suggestion
    void register_userUnderMinAge_notOk() {
```

what if min age will change in future